diff --git a/00main b/00main
index 8a9b520..f2e51a3 100644
--- a/00main
+++ b/00main
@@ -35,6 +35,7 @@ CARDINFO{driver}=="CA0106", INCLUDE="ca0106", GOTO="init_end"
 CARDINFO{driver}=="ENS1371", INCLUDE="ens1371", GOTO="init_end"
 CARDINFO{driver}=="rockchip_rt5616", INCLUDE="rt5616", GOTO="init_end"
 CARDINFO{driver}=="rockchip_rt5640", INCLUDE="rt5640", GOTO="init_end"
+CARDINFO{driver}=="realtek_rt5651-", INCLUDE="rt5651", GOTO="init_end"
 CARDINFO{driver}=="Test", INCLUDE="test", GOTO="init_end"
 
 LABEL="init_end"
